### ISSUE TITLE
复现结果与论文中的有差距

### ISSUE THREAD

**Issue by sdgixfg** [crotonyl/HETT#6]
### 问题概述

用 Dropbox 发布的 `best_val_unseen` 权重跑 eval，test_unseen SR 只有 ~26%，而论文 AAAI 2026 报告的 HETT* 在 refined CityNav 上是 **28.90%**。排除了代码、数据、环境等所有可能性，差距应该来自权重文件本身。

### 环境

- 代码：main 分支（commit `51eb86c`，与 `origin/main` 完全一致）
- 权重：Dropbox 下载的 `best_val_unseen`（MD5: `06ed6564a3f944155b57f21ee6e075a1`）
- 数据：Dropbox 下载的 refined CityNav（4 个 JSON 文件全部 MD5 校验通过）
- YOLO 权重/配置：MD5 校验通过
- PyTorch 2.12.0+cu130，GPU RTX 5090（也在 PyTorch 2.2.2 + A6000 上测过，结果相同）

### 复现结果

| 指标 | 论文 HETT* | 实测（作者权重） | 差距 |
|------|:---:|:---:|:---:|
| val_seen SR | 31.09 | 30.08 | -1.01 |
| val_unseen SR | 19.10 | 17.95 | -1.15 |
| **test_unseen SR** | **28.90** | **26.04** | **-2.86** |
| test_unseen SPL | 23.79 | 21.13 | -2.66 |

多次运行结果稳定在 test_unseen 25.9-26.3%。

### 已排除的因素

- [x] 代码与 GitHub 一致（main = origin/main）
- [x] 权重 MD5 与 Dropbox 一致
- [x] 全部 4 个数据文件 MD5 与 Dropbox 一致
- [x] YOLO 权重/配置 MD5 与 Dropbox 一致
- [x] eval 命令与 eval.sh 功能等价
- [x] 模型加载时 ALL KEYS matched（无缺失或多余参数）
- [x] eval 时已调用 model.eval()
- [x] grid_size=5, seed=0, success_dist=20, max_action_len=20, move_iteration=10
- [x] 多次运行结果一致
- [x] 重新从 Dropbox 下载权重后 MD5 完全一致，结果相同

### 疑问

1. Dropbox 里的权重就是论文 Table 1 报告 28.90 的那份吗？
2. 如果不是，能否 release 对应 28.90 的那份权重？
3. 是否有代码/config 之外的训练或评测细节可以解释这个差距？

谢谢！

**Comment by Criskly** [comment]
<img width="1009" height="99" alt="Image" src="https://github.com/user-attachments/assets/be300b91-9b7d-4cda-87cb-a8592701db41" />
我也是，结果更低
